### PR TITLE
🌱 add hack/verify-release.sh

### PR DESCRIPTION
**Pull request by tuminoid**
Manual cherry-pick of #339 without releasing.md parts.

It is identifying quite of few module issues on 1.4 branch btw. I will fix them separately.

### CODE REVIEW

**Comment by tuminoid** [comment]
/override test-ubuntu-e2e-integration-release-1-4 

Save the trees.

**Review by lentzi90 — COMMENTED** [review]
/approve

File: hack/verify-release.sh [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+        # shellcheck disable=SC2076
+        if ! [[ "$(jq .body "${RELEASE_JSON}")" =~ "${string}" ]]; then
+            echo "WARNING: '${string}' not found in release note text, recheck content"
+        fi
+    done
+
+    echo -e "Done\n"
+}
+
+verify_release_artefacts()
```
Comment (kashifest) [inline hack/verify-release.sh:351]: artifacts

File: hack/verify-release.sh [hunk 2]
```diff
@@ -0,0 +1,574 @@
...
+#
+
+# git tags
+declare -a git_annotated_tags=(
+    "v${VERSION}"
+)
+
+declare -a git_lightweight_tags=(
+    "api/v${VERSION}"
+    "test/v${VERSION}"
```
Comment (kashifest) [inline hack/verify-release.sh:74]: We don't have test module in IPAM ?

**Review by kashifest — COMMENTED** [review]
Questions/nits inline.

File: hack/verify-release.sh (hunk 1 above)
Comment (kashifest) [reply hack/verify-release.sh:351]: Quite a few of this in the file, you can fix all

File: hack/verify-release.sh (hunk 2 above)
Comment (tuminoid) [reply hack/verify-release.sh:74]: True. We had `test` but it is only containing fuzz. Removed.

File: hack/verify-release.sh (hunk 1 above)
Comment (tuminoid) [reply hack/verify-release.sh:351]: Artefact is British English spelling, artifact is just Americans mangling proper English :)

Comment (kashifest) [reply hack/verify-release.sh:351]: Ah, true. I am more used to artifact , in my knowledge base artefact never existed and just seemed wrong 😄

**Pull request merged**